Add prettier

agnesesterhuizen opened the pull request:
Hey Ian! I've added prettier with some basic rules and a npm script to run it on the source and example code. I have also added it to run with the npm prepare script but I wasn't completely sure if this is the right stage to run it. Please let me know if you think there's anything I should change or add or if there are any other issues. 

Fixes #3

Commit:
add prettier

ianobermiller commented on .prettierrc:
```diff
@@ -0,0 +1,7 @@
+  "tabWidth": 2,
+  "singleQuote": true,
+  "trailingComma": "all"
+}
```
```
{
  "bracketSpacing": false,
  "jsxBracketSameLine": true,
  "printWidth": 80,
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "all",
  "useTabs": false
}
```

ianobermiller replied:
Let's also add the rest of the config options. In particular I don't want `bracketSpacing` :)

ianobermiller commented on package.json:
```diff
@@ -15,9 +15,10 @@
     "build": "rollup -c",
     "start": "rollup -c -w",
-    "prepare": "yarn run build",
+    "prepare": "yarn run prettier && yarn run build",
```
lgtm

ianobermiller reviewed (changes requested):
Thanks for picking up this task! Sorry for the churn! Just fix the config and rebase and you should be good to go!

Commit:
Merge branch 'master' into add-prettier

agnesesterhuizen commented:
@ianobermiller Sorry for the delay in working on this, I've updated the config now and fixed the merge conflicts. Please let me know if theres anything else you think needs to change.

ianobermiller commented:
No worries, it's open source, no rush! Thanks for taking this on and fixing it up!
